src/providers/ipa/ipa_views.c

```diff
@@ -386,11 +386,13 @@ struct ipa_get_trusted_override_state {
     const char *ipa_realm;
     const char *ipa_view_name;
     struct dp_id_data *ar;
+    struct sss_domain_info *sdom;
```

Comment:
Hi,

typically `sdom` is used as a variable name for a `struct sdap_domain` object, so I found the use of `sdom` here a bit confusing, would it be possible to find a different name?

bye,
Sumit